tempesta_fw/cache.c

```diff
@@ -1402,7 +1401,7 @@ static TfwCfgSpec tfw_cache_cfg_specs[] = {
 };
 
 TfwCfgMod tfw_cache_cfg_mod = {
-	.name 	= "cache",
+	.name = "cache",
```

Comment:
The motivation here was to have the right part of the assignment aligned. That's broken now with this change. It was aligned before.